.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -19,11 +19,17 @@ If you haven't already done so, sign the Contributor License Agreement (CLA) as
 <!--
 Provide a brief description of the feature/change you are planning on implementing.
 -->
+Link to PRD: 
+<!--
+Add a link to the PRD (make sure it is accessible by anyone) here.
```

Comment:
It would be useful to explain what a PRD is, when it's required, and link to an example document, here. It's a pretu high bar, otherwise.

Reply:
Taken a stab at this inline.